docs/millennium-os/manual/chapters/30_usage.md

```diff
@@ -1,32 +1,32 @@
+At this point, you might have already gained a feeling for how MillenniumOS is designed to work - the Configuration Wizard that starts up after installation *uses* MillenniumOS tool-change and probing procedures to gather the necessary settings, information, and measurements to effectively probe work pieces.
 
-That said, there are a number of different probing methods, and the post-processor is not used during the configuration process so it is important to explain how the post-processor and MillenniumOS on RRF interact.
+That said, there are several different probing methods, and the post-processor is not used during the configuration process, so it is important to explain how the post-processor and MillenniumOS on RRF interact.
 
 ## Post-Processor
 
-The MillenniumOS post-processors are relatively simple - rather than outputting any particularly complex gcode sequences to perform tool changes or probing cycles, they rely on the knowledge that the firmware (RRF, and therefore MillenniumOS macros running on top of it) are best-placed to implement these in a safe manner. The gcode is designed to be simple to read and well commented, so for those of us of a less-trusting nature, it should be easy enough to open in a text editor and understand what it will command your mill to do.
+The MillenniumOS post-processors are relatively simple - rather than outputting any particularly complex gcode sequences to perform tool changes or probing cycles, they rely on the knowledge that the firmware (RRF, and therefore MillenniumOS macros running on top of it) are best placed to implement these safely. The gcode is designed to be simple to read and well commented, so for those with a less-trusting nature, it should be easy enough to open in a text editor and understand what it will command your mill to do.
 
 ![Image showing Post-Process tab of a Fusion360 Setup](../img/mos_usage_step_0.png){: .shadow-dark }
 
 ---
 
 ### WCS Offsets / Work Co-ordinate Systems
 
-The **"WCS offset"** setting of the **"Setup"** selects the WCS that will be switched into prior to running the tool paths configured in the Setup.
+The **"WCS offset"** setting of the **"Setup"** selects the WCS that will be switched into before running the tool paths configured in the Setup.
 
-A value of `0` in this field corresponds to `G54`, or the *first* WCS. We do not allow output of gcode in machine co-ordinates so this field is zero-indexed. A value of 2 corresponds to `G56`, or the *third* WCS.
+A value of `0` in this field corresponds to `G54`, or the *first* WCS. We do not allow the output of gcode in machine coordinates so this field is zero-indexed. A value of 2 corresponds to `G56`, or the *third* WCS.
```

Comment:
This behaviour was changed in `v0.3.0` to follow industry standard.

```suggestion
A value of `0` or `1` in this field corresponds to `G54`, or the *first* WCS, as we do not allow the output of gcode in machine coordinates (`G53`). 

This follows industry standard so that offsets 2 and 3 correspond to `G55` and `G56` respectively.
```